openlane/scripts/odbpy/disconnected_pins.py

```diff
@@ -278,6 +277,7 @@ def main(
         rich.print(critical_table)
     if full_table.row_count > 0:
         if full_table_path := write_full_table_to:
+            full_table.width = 160
```

Comment:
why not `min_width`, for a table that needs > 160 width?

Reply:
good point